src/ui/MessageItemMenu/index.tsx

```diff
@@ -44,8 +44,8 @@ export default function MessageItemMenu({
   const showMenuItemCopy: boolean = isUserMessage(message as UserMessage);
   const showMenuItemReply: boolean = false && !isFailedMessage(channel, message) && !isPendingMessage(channel, message);
   const showMenuItemEdit: boolean = (isUserMessage(message as UserMessage) && isSentMessage(channel, message) && isByMe);
-  const showMenuItemResend: boolean = (isFailedMessage(channel, message) && message.isResendable() && isByMe);
-  const showMenuItemDelete: boolean = (isSentMessage(channel, message) && isByMe);
+  const showMenuItemResend: boolean = (isFailedMessage(channel, message) && message?.isResendable?.() && isByMe);
+  const showMenuItemDelete: boolean = isByMe;
```

Comment:
isSentMessage(channel, message) -> No errors while deleting unsent message?

Reply:
There was no new appeared error when I remove this condition